core/src/main/java/com/klaytn/caver/wallet/KeyringContainer.java

```diff
@@ -235,7 +345,13 @@ public MessageSigned signMessage(String address, String data, int role, int inde
     }
 
     /**
-     * Signs the transaction using all keys in the Keyring instance corresponding to the address.
+     * Signs the transaction using all keys in the Keyring instance corresponding to the address.<p>
```

Comment:
```suggestion
     * Signs the transaction as a sender of the transaction and appends signatures in the transaction instance using all keys in the Keyring instance corresponding to the address.<p>
```

FYI. I referenced [js docs](https://docs.klaytn.com/dapp/sdk/caver-js/api-references/caver.wallet#caver-wallet-sign).